### ISSUE TITLE
question

### ISSUE THREAD

**Issue by FrozenStorm1029** [jason0611/FS22_DashboardLive#85]
are all base game vehicles prepared with this mod

**Comment by jason0611** [comment]
No, only a few in-game vehicles get DBL inserted. Please look out for mods with DBL on ModHub or similar places.

**Comment by FrozenStorm1029** [comment]
is there any base game combine that has been prepared?

**Comment by jason0611** [comment]
No, there isn't. But one of the best DBL-integrated combines can be found here: https://sharemods.com/o746tv06v6bw/FS22_Claas_Lexion_Pack.zip.html

I know, that Sharemods is a terrible site regarding Ads and misleading links, but the Claas Lexion Pack is worth the effort.
